app/mailers/organizer_position_invites_mailer.rb

```diff
@@ -4,18 +4,24 @@ class OrganizerPositionInvitesMailer < ApplicationMailer
   before_action :set_invite
 
   def notify
-    mail to: @invite.user.email_address_with_name, subject: @invite.initial? && @invite.event.demo_mode? ? "Thanks for applying for HCB 🚀" : "You've been invited to join #{@invite.event.name} on HCB 🚀"
+    # if somebody gets invited with a link, removed, and then invited regularly they wont get an email here
+    # we dont currently assosciate OPI and OPIR / OPIL
```

Comment:
Ah, we should associate OPIR and OPI to fix this. Add a `belongs_to: :organizer_position_invite` to OPIR.

This likely means that `OrganizerPositionInviteService::Create` should take in an optional `organizer_position_invite_request` argument.